compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUTensorPad.cpp

```diff
@@ -47,8 +49,12 @@ static FailureOr<SmallVector<Value>> rewriteAsPaddedOp(
     SmallVector<int64_t> paddedShape(shape.begin(), shape.end());
     for (const auto &en : llvm::enumerate(tensorLoad.getMixedSizes())) {
       if (en.value().is<Attribute>()) {
-        paddedShape[en.index()] =
-            en.value().get<Attribute>().dyn_cast<IntegerAttr>().getInt();
+        Optional<int64_t> cst = getConstantIntValue(en.value());
+        if (!cst) {
+          return rewriter.notifyMatchFailure(
+              linalgOp, "No constant bounding box can be found for padding");
+        }
+        paddedShape[en.index()] = cst.getValue();
       } else {
```

Comment:
We don't need the first check (i.e., is<Attribute>) if we're using getConstantIntValue method, right?